db/migrate/20200327164455_add_mark_remove_column_to_recents.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+class AddMarkRemoveColumnToRecents < ActiveRecord::Migration
+  disable_ddl_transaction!
+  def up
+    add_column :recents, :mark_remove, :boolean
```

Comment:
Clear the `STATEMENT_TIMEOUT=0` for running migration tasks (ENV var) or manually here to ensure none of these queries time out and break the migration.